espnet/mt/pytorch_backend/mt.py

```diff
@@ -183,7 +183,7 @@ def train(args):
                           batch_frames_in=args.batch_frames_in,
                           batch_frames_out=args.batch_frames_out,
                           batch_frames_inout=args.batch_frames_inout,
-                          mt=True)
+                          mt=True, iaxis=1, oaxis=0)
```

Comment:
You can remove `, iaxis=1, oaxis=0`.

Reply:
I'm afraid it is necessary because I removed the assignment in [line1](https://github.com/espnet/espnet/blob/f7d2100979f1e4f2e8d725bb1af3c67c1ad20271/espnet/utils/training/batchfy.py#L305) and [line2](https://github.com/espnet/espnet/blob/f7d2100979f1e4f2e8d725bb1af3c67c1ad20271/espnet/utils/training/batchfy.py#L320).

Reply:
You're right.